projects/sport-data-valley/seeds/questionnaire_helpers/complaints.rb

```diff
@@ -418,5 +422,23 @@ def all_complaint_options(with_otherwise = false)
       end
       result
     end
+
+    def all_complaint_scores(with_otherwise = false)
```

Comment:
shouldn't this be 'compliance'? (same for the function below)

Reply:
No, these are about complaints people have (as in physical ailments). So they are literally "complaint scores".